tools/topology/topology2/cavs/platform/intel/hdmi-generic.conf

```diff
@@ -0,0 +1,152 @@
+Object.Dai {
+	HDA.4 {
+		name iDisp1
+		id $HDMI0_ID
+		default_hw_conf_id 1
+		Object.Base.hw_config.HDA4 {}
+		direction playback
+	}
+	HDA.5 {
+		name iDisp2
+		id $HDMI1_ID
+		default_hw_conf_id 2
+		Object.Base.hw_config.HDA5 {}
+		direction playback
+	}
+	HDA.6 {
+		name iDisp3
+		id $HDMI2_ID
+		default_hw_conf_id 3
+		Object.Base.hw_config.HDA6 {}
+		direction playback
+	}
+}
+Object.Pipeline {
+	passthrough-playback.20 {
+		Object.Widget.pipeline.1 {
+			stream_name $HDMI0_PIPELINE_STREAM_NAME
+		}
+		Object.Widget.copier.1 {
+			stream_name $HDMI0_PCM_CAPS
+		}
+
+		format s16le
+		index $HDMI0_HOST_PIPELINE_ID
+	}
+	passthrough-be.20 {
+		direction	"playback"
+		index $HDMI0_DAI_PIPELINE_ID
+		Object.Widget.pipeline.1 {
+			stream_name $HDMI0_PIPELINE_STREAM_NAME
+		}
+
+		Object.Widget.copier.1 {
+			stream_name "iDisp1"
+			dai_type "HDA"
+			copier_type "HDA"
+		}
+		format s16le
+	}
+	passthrough-playback.21 {
+		Object.Widget.pipeline.1 {
+			stream_name $HDMI1_PIPELINE_STREAM_NAME
+		}
+		Object.Widget.copier.1 {
+			stream_name $HDMI1_PCM_CAPS
+		}
+
+		format s16le
+		index $HDMI1_HOST_PIPELINE_ID
+	}
+	passthrough-be.21 {
+		direction	"playback"
+		index $HDMI1_DAI_PIPELINE_ID
+		Object.Widget.pipeline.1 {
+			stream_name $HDMI1_PIPELINE_STREAM_NAME
+		}
+
+		Object.Widget.copier.1 {
+			stream_name 'iDisp2'
+			dai_type "HDA"
+			copier_type "HDA"
+		}
+		format s16le
+	}
+	passthrough-playback.22 {
+		Object.Widget.pipeline.1 {
+			stream_name $HDMI2_PIPELINE_STREAM_NAME
+		}
+		Object.Widget.copier.1 {
+			stream_name $HDMI2_PCM_CAPS
+		}
+
+		format s16le
+		index $HDMI2_HOST_PIPELINE_ID
+	}
+	passthrough-be.22 {
+		direction	"playback"
+		index $HDMI2_DAI_PIPELINE_ID
+		Object.Widget.pipeline.1 {
+			stream_name $HDMI2_PIPELINE_STREAM_NAME
+		}
+
+		Object.Widget.copier.1 {
+			stream_name 'iDisp3'
+			dai_type "HDA"
+			copier_type "HDA"
+		}
+		format s16le
+	}
+}
+Object.PCM {
+	pcm.5 {
+		name HDMI1
+		id 5
+		Object.Base.fe_dai.HDMI1 {}
+		Object.PCM.pcm_caps.playback {
+			name $HDMI0_PCM_CAPS
```

Comment:
HDMI1 -> HDMI0_PCM_CAPS ? better the same name

Reply:
> HDMI1 -> HDMI0_PCM_CAPS ? better the same name

ack

Reply:
> HDMI1 -> HDMI0_PCM_CAPS ? better the same name

Updated.